api/src/main/resources/fireCloud.yaml

```diff
@@ -371,6 +371,21 @@ paths:
       parameters:
         - $ref: '#/parameters/workspaceNamespaceParam'
         - $ref: '#/parameters/workspaceNameParam'
+        - name: fields
```

Comment:
Awesome updates! Let's see if we can move away from keeping this file in our source control.

Reply:
Agreed! I'm glad to hear there's a ticket for that work. (David An gets credit for this swagger documentation, its fabulous and I just copied what he did.)